src/main/java/com/powsybl/openloadflow/network/impl/LfBranchImpl.java

```diff
@@ -205,6 +203,30 @@ public double getPermanentLimit2() {
         return branch.getCurrentLimits2() != null ? branch.getCurrentLimits2().getPermanentLimit() * getBus2().getNominalV() / PerUnit.SB : Double.NaN;
     }
 
+    @Override
+    public Map<Integer, Double> getTemporaryLimits1() {
+        Map<Integer, Double> map = new HashMap<>();
```

Comment:
This code is duplicated in all classes extending LfBranch, shouldn't we add a `getTemporaryLimits(CurrentLimits currentLimits, double nominalVoltage)` method in `AbstractLfBranch`?